# Fix for LSB compliance. Heartbeat relies on LSB behvaiour for correct high-availability
--- old/dhcpd	2009-10-02 16:13:53.000000000 -0500
+++ dhcpd	2009-10-02 16:17:25.000000000 -0500
@@ -89,7 +89,8 @@
 stop() {
     pidofproc $prog >/dev/null 2>&1
     if [ $? -ne 0 ]; then
-        RETVAL=7
+        # dhcpd is not running; return 0 (fix for LSB compliance)
+        RETVAL=0
         return $RETVAL
     fi
 
